### PR TITLE
Check off todo list from readme file and migrate functions from hyperSpec to hpSpc.read.spe 

### PR DESCRIPTION
**Pull request by sangttruong**

### CODE REVIEW

**Review by bryanhanson — CHANGES_REQUESTED** [review]
1.  I should not be an author on this package, I haven't contibuted anything and I don't know anything about this format!
2. I don't  recall what exactly Erick contributed to this last year but he should remain as an author, or maybe a contributor.
3. Re: the copyright holder, please stick to the language in the template.  All the authors jointly hold the copyright.
4. For `NEWS.md` you should keep the Acknowledgments section and it should thank GSOC for supporting both you and Erick.
5. `project.Rproj` should be git-ignored, this is local to your computer

**Review by bryanhanson — APPROVED** [review]
Looks Good.

**Review by bryanhanson — COMMENTED** [review]
I think you need to remove the comma at the end of line 8.

File: R/wl_convert_units.R [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+#' @param ref_wl laser wavelength (required for work with Raman shift)
+#' @author R. Kiselev
+#' @export
+#'
+#' @concept wavelengths
+#'
+#' @examples
+#' wl_convert_units(3200, "Raman shift", "nm", ref_wl = 785.04)
+#' wl_convert_units(785, "nm", "invcm")
+wl_convert_units <- function(x, from, to, ref_wl = NULL) {
```
Comment (GegznaV) [inline R/wl_convert_units.R:19]: @sangttruong, @bryanhanson  Do I understand correctly, that this file `wl_convert_units.R` was moved/copied from `hyperSpec` to `hySpc.read.spe`? What is the purpose to do so? And why copying/moving was preferred over exporting?

File: R/0temp_fileio_optional.R [hunk 2]
```diff
@@ -0,0 +1 @@
+.spc_io_postprocess_optional <- hyperSpec::.spc_io_postprocess_optional
```
Comment (GegznaV) [inline R/0temp_fileio_optional.R:1]: @sangttruong Why is the mechanism of copying a function was preferred over exporting?

File: DESCRIPTION [hunk 3]
```diff
@@ -1,14 +1,14 @@
-Package: SKELETON
+Package: hySpc.read.spe
 Type: Package
 Version: 0.0.0.9000
-Title: Package's Title
-Description: Short Description of the package.
+Title: File Import Functions for Spectra produced by Princeton Instruments spectrometers
```
Comment (GegznaV) [inline DESCRIPTION:4]: Title case must be used. Otherwise CRAN tests will fail.

File: NEWS.md [hunk 4]
```diff
@@ -1,15 +1,16 @@
...
+## User-Visible Changes
 
-## User-visible Changes
-
-### Bug Fixes
-### Breaking Changes
-### New Functions
-
-## Internal Changes
+* Moved functions from **hyperSpec** to **hySpc.read.spe**:
```
Comment (GegznaV) [inline NEWS.md:9]: 1. Were the functions properly deprecated in `hyperSpec` package?

2. Was the statement that some functions were deprecated and their analogs can be found in `hySpc.read.spe` package included on `NEWS.md` of `hyperSpec` package?

File: DESCRIPTION [hunk 5]
```diff
@@ -1,14 +1,14 @@
...
 Authors@R: c(
     person("Claudia",   "Beleites",  role = c("aut"), email = "Claudia.Beleites@chemometrix.gmbh"),
```
Comment (GegznaV) [inline DESCRIPTION:6]: I think, several more people (incl. Roman Kiselev) must be included in the list of authors.

File: NEWS.md [hunk 6]
```diff
@@ -1,15 +1,16 @@
-# SKELETON
+# hySpc.read.spe 0.0.x 2021-xx-xx
 
 ## Acknowledgements
-The `hyperSpec` team gratefully acknowledges support from the Google Summer of Code program, which sponsored student Erick Oduniyi during summer 2020.
-Erick and the team carried out a significant overhaul of `hyperSpec` which led to this release.
+The `hyperSpec` team gratefully acknowledges support from the Google Summer of Code program, which sponsored student Erick Oduniyi during Summer 2020 and Sang Truong during summer 2021.
```
Comment (GegznaV) [inline NEWS.md:4]: Did Erick work on this package last year?

File: R/read_spe.R [hunk 7]
```diff
@@ -0,0 +1,425 @@
+# Reading of SPE files, produced by Princeton Instruments spectrometers
+# File format version 2.5 (Sept. 2002)
+
+# C. Beleites
+# R. Kiselev
```
Comment (GegznaV) [inline R/read_spe.R:5]: It is clear, that Roman Kiselev (@ximeg) must be included among the authors of this package.

File: R/wl_convert_units.R (hunk 1 above)
Comment (GegznaV) [reply R/wl_convert_units.R:19]: I think this function must be left in `hyperSpec` and imported here using `#' @importFrom hyperSpec  wl_convert_units`.  I'll add a separate comment where is a good place to add this `roxygen2` comment.

We may discuss this if needed.

File: R/package.R [hunk 8]
```diff
@@ -2,7 +2,9 @@
 #'
 #' @import hyperSpec
 #' @import hySpc.testthat
-
+#'
+#' @importFrom methods is
+#' @importFrom utils maintainer
 "_PACKAGE"
```
Comment (GegznaV) [inline R/package.R:8]: This file (between lines 12 and 13, i.e., between `## usethis namespace: start`  and `## usethis namespace: end`) is a good place to add `roxygen2` comment that I mentioned in some other comment:
`#' @importFrom hyperSpec  wl_convert_units`

Please, make sure that you use special comment `#'` and not `##'`

**Review by GegznaV — COMMENTED** [review]
Even though this branch is merged, please, address the issues I raised in a new PR or discuss them.

File: NEWS.md (hunk 6 above)
Comment (sangttruong) [reply NEWS.md:4]: I'm not sure. @eoduniyi Did you work on this last summer? I mention @bryanhanson here too since he might know.

Comment (eoduniyi) [reply NEWS.md:4]: Hello! I did not work on this package 🙂:

File: DESCRIPTION (hunk 5 above)
Comment (sangttruong) [reply DESCRIPTION:6]: I add Roman, but I'm not sure who else should I add.

File: R/read_spe.R (hunk 7 above)
Comment (sangttruong) [reply R/read_spe.R:5]: Yes, I added him to the description.md file.

File: NEWS.md (hunk 6 above)
Comment (sangttruong) [reply NEWS.md:4]: Thank you for letting me know!

File: R/wl_convert_units.R (hunk 1 above)
Comment (GegznaV) [reply R/wl_convert_units.R:19]: Addressed in #3

File: DESCRIPTION (hunk 3 above)
Comment (GegznaV) [reply DESCRIPTION:4]: Addressed in #3

File: DESCRIPTION (hunk 5 above)
Comment (GegznaV) [reply DESCRIPTION:6]: I think we should ask @cbeleites to be totally sure. I opened #4 for the discussion so we will be able to fix this in the future (after the issue is discussed). So after my notes are addressed in #3, I think we can merge #3.

File: R/0temp_fileio_optional.R (hunk 2 above)
Comment (GegznaV) [reply R/0temp_fileio_optional.R:1]: Addressed in #3

File: R/package.R (hunk 8 above)
Comment (GegznaV) [reply R/package.R:8]: Addressed in #3

File: R/wl_convert_units.R (hunk 1 above)
Comment (sangttruong) [reply R/wl_convert_units.R:19]: @bryanhanson

**Pull request merged**